website2/docs/topology-development-topology-api-python.md

```diff
@@ -687,8 +686,7 @@ $ heron submit local \
 Note the `-` in this submission command. If you define a topology by subclassing `TopologyBuilder` you do not need to instruct Heron where your main method is located.
 
 > #### Example topologies buildable as PEXs
-> * See [this repo](https://github.com/streamlio/pants-dev-environment) for an example of a Heron topology written in Python and deployable as a Pants-packaged PEX.
```

Comment:
this even appears to be a duplicate within the same file (given the diff is the same as the hunk above)

Reply:
Addressed.